website_signup_legal_page_required/views/pages.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
add copyright xml files too as in https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml